logstreamer/filehandling.go

```diff
@@ -471,7 +475,11 @@ func (ls *LogstreamSet) ScanForLogstreams() (result []string, errors *MultipleEr
 			if err != nil {
 				errors.AddMessage(err.Error())
 				position.Reset()
+				initialTail = ls.initialTail
+			} else if position.SeekPosition == 0 {
```

Comment:
Yes, you'd want to check for empty `Filename` and `Hash` attributes as well. I'd add an `IsZero()` method to the LogstreamLocation struct and that checks all three values and use `else if position.IsZero()` instead.